packages/integration/src/gitlab/core.test.ts

```diff
@@ -121,6 +131,12 @@ describe('gitlab core', () => {
         result:
           'https://gitlab.com/api/v4/projects/12345/repository/files/my%2Fpath%2Fto%2Ffile.yml/raw?ref=branch',
       },
+      {
+        config: configWithSelfHosted,
+        url: 'https://gitlab.com/gitlab/groupA/teams/teamA/subgroupA/repoA/-/blob/branch/my/path/to/file.yml',
```

Comment:
i thought the gitlab.com case was excluded above, maybe i'm confused but it's surprising that this would be gitlab.com instead of g.com for example - maybe I'm reading the test wrong